### DISCUSSION TITLE
Warning note for external uploads

### DISCUSSION THREAD

**Discussion by aviv926** [immich-app/immich#12596] [Feature Request]
### The bug

I'm not sure if this is a feature suggestion or a "bug".
But when someone with an external link uploads a file to shared albume, they can't delete that file after uploading.

Maybe we should add a warning message for such cases.

### The OS that Immich Server is running on

Main Demo (1.98.2)

### Version of Immich Server

Main Demo (1.98.2)

### Version of Immich Mobile App

Main Demo (1.98.2)

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Main Demo (1.98.2)
```

### Your .env content

```Shell
Main Demo (1.98.2)
```

### Reproduction steps

```bash
1.Upload a file to the album with a link as an unregistered user
```

**Comment by jrasm91** [comment]
Thanks for bringing this up. Maybe in the future we can look at adding a warning message specifically for shared links. Since this is more of an enhancement around anonymous uploads, I'm going to convert it to a discussion instead.

**Comment by Menghini** [comment]
Along with the feature, I think it would also be beneficial to work #10150 alongside this. It would be nice to be notified when there are photos/videos added anonymously to an album so I can police it easier.
